{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/version.py

```diff
@@ -7,4 +7,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = '1.0.0'
+__version__ = '1.0.0.dev{{ cookiecutter.year }}0000'
```

Comment:
Since you anyway have to edit above license file, can you leave this as ``1.0.0``. It's because for modules we use the dev approach for an instance, we don't normally use it.